src/get-sysroot.sh

```diff
@@ -66,7 +66,9 @@ case "$ARCH" in
     WITH_GN=windows
     case "$target_cpu" in
       x64) WITH_PGO=win64;;
-      *) WITH_PGO=win32;;
+      x86) WITH_PGO=win32;;
+      arm64) WITH_PGO=win32;;
+      *) WITH_PGO=win64;;
```

Comment:
what is the purpose of this?

Reply:
It will use the correct target_cpu in native compilation (for windows, it is always x64) if not specified. For cross-compilation, it uses x86 for arm64  (tested).

Reply:
No, what is the purpose of this being different from Chromium upstream?

Reply:
it doesn't  change anything in code. I checked the map it should be matched. Without this change, build.sh doesn't work in a windows building environment (it sticks to win32 pgo while compiler perfers to win64 pgo).

If you dont like it, I can upstream it later.

Reply:
> compiler perfers to win64 pgo

Why specifically?

Reply:
If target_cpu is not specified (for native compiling, it is win64 for build machine), it will be pointed to win32.

Reply:
```bash
if [ ! "$target_cpu" -a $(uname -m) = x86_64 ]; then
  target_cpu=x64
fi
```